mx-rust-semantics/main/preprocessing/proxy.md

```diff
@@ -0,0 +1,61 @@
+```k
+
+module MX-RUST-PREPROCESSING-PROXY
+    imports COMMON-K-CELL
+    imports MX-RUST-REPRESENTATION
+    imports RUST-PREPROCESSING-CONFIGURATION
+
+    rule
+        <k>
+            rustMxAddProxyMethods(Trait:TypePath)
+            => error
+                ( "execute_on_dest_context already exists for trait"
+                , ListItem(Trait)
+                )
+            ...
+        </k>
+        <trait-path> Trait </trait-path>
+        <method-name> #token("execute_on_dest_context", "Identifier") </method-name>
+        [priority(50)]
```

Comment:
50 is the default priority of every rule. I don't think this line is needed.

Reply:
I think this priority number is used in other parts of the `mx-rust-semantics`, such as in `mx-rust-semantics/main/preprocessing/methods.md`, but in previous commits.

Reply:
I know, but I am using this to make the relation with the rule below more obvious. Maybe this is too much, so I'll send a cleanup PR if you think it should be removed.